.github/workflows/benchmarks.yml

```diff
@@ -22,9 +22,9 @@ on:
             type: boolean
             description: "Boolean to enable the Linux 5.10 Swift version matrix job. Defaults to true."
             default: true
-        linux_nightly_6_0_enabled:
+        linux_6_0_enabled:
```

Comment:
Bad news: If there are other repos who call these reusable workflows and do use these parameters, their CIs will start to fail.

Reply:
That is intentional. This is where we landed when we wanted to achieve both an easy way to roll out new Swift versions but also give the repos the opportunity to define their own custom arguments that are passed.

Reply:
I confirm Swift Runtime for AWS Lambda CI failed.  Fixed now :-)